### DISCUSSION TITLE
Looking for a more contributions with The Graph on Kaia

### DISCUSSION THREAD

**Discussion by zxstim** [kaiachain/kaia-dapp-mono#203] [Requests]
So Kaia just got [The Graph](https://thegraph.com/) integration for indexing super boost.

That means I am looking for:
- indexing kits,
- guides,
- tutorials,
- or build cool things with the graph on Kaia

pitch your ideas for approval here: https://dub.sh/build-on-kaia

**Comment by anudeep11k** [comment]
Hey zxtsim check this out 
https://github.com/kaiachain/kaia-dapp-mono/issues/212

**Reply by zxstim** [reply]
hey, replied. I need to get used to the graph more before I can assess your submission. Feel free to propose and build something new for the time being.
